device-types/Cisco/MDS-C9148-16P.yaml

```diff
@@ -0,0 +1,137 @@
+---
+manufacturer: Cisco
+model: MDS 9148 16P
```

Comment:
The model should be the same as the one on the device, either the one on the front of the device or the one that the configuration shows. It appears the front of the devices shows `DS-C9148-16P-K9`, unless the `show version` or other applicable command shows something different please change this accordingly

Reply:
The front of the device has DS-C9148-16p-K9 on it, however the show version command display shows the following "Hardware cisco MDS 9148 FC (1 Slot) Chassis ("1/2/4/8 Gbps FC/Supervisor-3")". I guess we should keep it the same as what is on the front of the device.